src/views/sidebar/components/tabs/summary-tab.js

```diff
@@ -105,7 +96,8 @@ export default class SummaryTab extends Component {
                         inputProps={{ placeholder: '' }}
                     />
                 </section>
-                <DescriptionBox {...descriptionProps} />
+                <span className='summary-tab-category'>Description</span>
+                <DescriptionBox {...this.props} />
```

Comment:
@tjboswell Does this bind all the props to the component, or just the props that `DescriptionBox` specifically lists in `this.propTypes`?